### PR TITLE
Add At.js docs to README

### PR DESCRIPTION
**Pull request by rexagod**
Part of #55.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by rexagod** [comment]
@jywarren @publiclab/reviewers

File: README.md [hunk 1]
```diff
@@ -102,7 +103,19 @@ Module output is collected (by `editor.collectData()`) in the `editor.data` obje
...
+
+- Set up multiple listeners for different characters with different behavior and data.
+- Format returned data using **custom templates**, that support keyboard and mouse imput.
+- Custom data handlers and template renderers using a group of configurable callbacks.
+
+We currently employ the `At.js` library to autocomplete authors, wiki pages, and emojis, by wiring them to the "@", "#", and ":" symbols respectively. Refer to the demonstration below for a better insight as to how this works.
+
+![atwho_js](https://user-images.githubusercontent.com/33557095/53129501-b54e9e00-358d-11e9-8ebb-555ca5a1fdcc.gif)
+
+Detailed documentation can be referred to at the [At.js wiki pages](https://github.com/ichord/At.js/wiki).
```
Comment (jywarren) [inline README.md]: This looks good; can we link to a demo of this running, and could we show some inline code examples for how to configure it to run with the editor? Thanks!!!

**Comment by rexagod** [comment]
https://github.com/publiclab/PublicLab.Editor/pull/293

**Comment by jywarren** [comment]
Awesome, thanks!!!

**Pull request merged**